### PR TITLE
Mykola/footer

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: dist/components/Second-block.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            </div>
+            <div class="body-color__item">
+                <img src="css/images/body-color4.jpg" alt="">
+            </div>
+            <div class="body-color__item">
+                <img src="css/images/body-color5.jpg" alt="">
+            </div>
+        </div>
+        <div class="wheels">
+            <div class="wheels__item first-item"></div>
```
Comment (StadniukYevhen) [inline dist/components/Second-block.html]: rewrite `BEM`

File: dist/components/Sixth-block.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            и интеллектуальное освещение дороги, повышая уровень безопасности и делая Вас заметным.
+            Светодиодные фары Audi Matrix эффектны не только ночью, но и в дневное время.
+        </p>
+    </div>
+    <div class="test-drive">Тест-драйв Новый Q7</div>
+    <button class="test-drive-button"><span>Запись</span></button>
+    <div class="test-drive-form">
+        <svg xmlns="http://www.w3.org/2000/svg" class="close-form" viewBox="0 0 17 16">
+            <defs>
+                <style>
```
Comment (StadniukYevhen) [inline dist/components/Sixth-block.html]: move to styles

File: dist/components/Second-block.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <circle class="inner-circle__outline" cx="100" cy="100" r="45.6" />
+            <path class="play" d="M65,46.5L39,63V30Z" transform="translate(55 55)" />
+            <path class="pause" d="M39,36h4V57H39V36Zm11,0h4V57H50V36Z" transform="translate(55 55)" />
+            <circle class="big-circle__outline" cx="100" cy="100" r="80" />
+        </svg>
+    </div>
+    <div class="controls">
+        <div class="body-color">
+            <div class="body-color__item">
+                <img src="css/images/body-color1.jpg" alt="">
```
Comment (StadniukYevhen) [inline dist/components/Second-block.html]: fill `alt` attribute

File: dist/components/Sixth-block.html (hunk 2 above)
Comment (nmelnychenko) [reply dist/components/Sixth-block.html]: done

File: dist/components/Second-block.html (hunk 3 above)
Comment (nmelnychenko) [reply dist/components/Second-block.html]: done

File: dist/components/Sixth-block.html (hunk 2 above)
Comment (nmelnychenko) [reply dist/components/Sixth-block.html]: done

**Pull request merged**